Bearded.Utilities.Tests/Collections/DeletableObjectListTests.cs

```diff
@@ -79,13 +77,20 @@ public void ThrowsForNegativeValues(int count)
                 createListWithNegativeValue.Should().Throw<ArgumentOutOfRangeException>();
             }
         }
-        
+
         public abstract class MethodThatDoesNotThrowsWhenEnumeratingTests
         {
             protected abstract void CallMethod(DeletableObjectList<TestDeletable> list);
 
-            public static IEnumerable<object[]> IndicesFromZeroToNineteen =
-                Enumerable.Range(0, 20).Select(i => new object[] {i});
+            public static TheoryData<int> IndicesFromZeroToNineteen = new TheoryData<int>();
+
+            static MethodThatDoesNotThrowsWhenEnumeratingTests()
```

Comment:
Use of this static constructor confused me for a moment. I’d rather use a collection initialiser above if you don’t mind, and turn it into a read only property as well (not sure why it wasn’t before?).

Reply:
Totally get that. I was on the fence too, but I forgot about making it a self-contained getter.